java/ql/lib/semmle/code/java/dataflow/FlowSources.qll

```diff
@@ -247,3 +247,17 @@ class ExportedAndroidIntentInput extends RemoteFlowSource, AndroidIntentInput {
 
   override string getSourceType() { result = "Exported Android intent source" }
 }
+
+/** A parameter of an entry-point method declared in a `ContentProvider` class. */
+class AndroidContentProviderInput extends DataFlow::Node {
+  AndroidContentProvider declaringType;
```

Comment:
This field is unbound.

Reply:
Good catch, when I used `sourceNode` I accidentally removed the code that bound  the field. Fixed in d5f675c.